assets/airports/eidw.json

```diff
@@ -1009,316 +1439,398 @@
-                ["ein", 12],
-                ["ein/long", 3],
-                ["ryr", 10],
-                ["stk",  7],
-                ["baw",  5],
-                ["bcy",  5],
-                ["bee",  3],
-                ["aal/long",  3],
-                ["aca/long",  2],
-                ["dal/long",  3],
-                ["ual/long",  3],
-                ["swr",  1],
-                ["ibe",  1]
+                ["bee",    5],
+                ["ein",   20],
+                ["fastga", 3],
+                ["nax",    3],
+                ["ryr",   30],
+                ["sas",    5],
+                ["stk",   15],
+                ["tom",    2]
             ]
         },
         {
             "origin": "EIDW",
             "destination": "",
             "category": "departure",
             "route": "EIDW.NEVRI.NEVRI",
-            "altitude": null,
-            "speed": null,
+            "altitude": "",
+            "speed": "",
             "method": "random",
-            "rate": 3.5,
+            "rate": 1,
             "airlines": [
-                ["ein", 12],
-                ["ein/long", 3],
-                ["ryr", 10],
-                ["stk",  7],
-                ["baw",  5],
-                ["bcy",  5],
-                ["bee",  3],
-                ["aal/long",  3],
-                ["aca/long",  2],
-                ["dal/long",  3],
-                ["ual/long",  3],
-                ["swr",  1],
-                ["ibe",  1]
+                ["aal/long", 5],
+                ["aca/long", 3],
+                ["bee",      5],
+                ["dal/long", 3],
+                ["ein",     20],
+                ["ein/long", 5],
+                ["eth/long", 3],
+                ["fastga",   3],
+                ["nax",      3],
+                ["ryr",     30],
+                ["sas",      5],
+                ["stk",     15],
+                ["ual/long", 3]
             ]
         },
         {
             "origin": "EIDW",
             "destination": "",
             "category": "departure",
             "route": "EIDW.PESIT.PESIT",
-            "altitude": null,
-            "speed": null,
+            "altitude": "",
+            "speed": "",
             "method": "random",
-            "rate": 3.5,
+            "rate": 4,
             "airlines": [
-                ["ein", 12],
-                ["ein/long", 3],
-                ["ryr", 10],
-                ["stk",  7],
-                ["baw",  5],
-                ["bcy",  5],
-                ["bee",  3],
-                ["aal/long",  3],
-                ["aca/long",  2],
-                ["dal/long",  3],
-                ["ual/long",  3],
-                ["swr",  1],
-                ["ibe",  1]
+                ["aea",    1],
+                ["aee",    1],
+                ["afr",    3],
+                ["baw",   10],
+                ["bcy",   10],
+                ["bee",    5],
+                ["ein",   20],
+                ["fastga", 3],
```

Comment:
this should be changed to `n/fastga`, as the `fastga` airline has been moved to a _fleet_ of the `n` airline.